core/zts/src/main/rdl/RoleCert.tdl

```diff
@@ -17,3 +17,8 @@ type RoleCertificateRequest Struct {
     Timestamp prevCertNotBefore (optional); //previous role certificate not before date
     Timestamp prevCertNotAfter (optional); //previous role certificate not after date
 }
+
+//The representation for an enumeration of roles
```

Comment:
Let's not create another object here. We already have one called RoleAccess with the same structure. That object is returned to indicate the list of roles the principal has access to in a domain and it's pretty much the same use.

Reply:
Moved RoleAccess from Token.rdli to RoleCert.tdl so it can be used by both (Token.rdli already includes RoleCert.tdl)